gnomad/variant_qc/random_forest.py

```diff
@@ -159,6 +159,93 @@ def get_columns_quantiles(
     return res
 
 
+def median_impute_features(
+    ht: hl.Table,
+    impute_features_by_variant_type: bool = False,
```

Comment:
should we make this an `expr` ? Seems like it'd make more sense now that this is going into a more generic lib.

Reply:
would also remove the need for that note